### ISSUE TITLE
the nan's problem has not been solved

### ISSUE THREAD

**Issue by a07913838438** [Bohao-Lee/CME#12]
Thanks for your great projects and Thanks for share the code of your great work!
I do this

    steps=-1,500,40000,60000
    scales=0.1,10,.1,.1

to

    steps=-1,500,40000,60000
    scales=0.01,100,.1,.1
but The problem remains,I don't know how to do it to solve this problem.Can you give me some advice?
![2021-08-24 10-39-11屏幕截图](https://user-images.githubusercontent.com/89434608/130547463-e09c0a32-7bb2-4277-b682-024eced4f399.png)

**Comment by Bohao-Lee** [comment]
> Thanks for your great projects and Thanks for share the code of your great work!
> I do this
> 
> ```
> steps=-1,500,40000,60000
> scales=0.1,10,.1,.1
> ```
> 
> to
> 
> ```
> steps=-1,500,40000,60000
> scales=0.01,100,.1,.1
> ```
> 
> but The problem remains,I don't know how to do it to solve this problem.Can you give me some advice?
> ![2021-08-24 10-39-11屏幕截图](https://user-images.githubusercontent.com/89434608/130547463-e09c0a32-7bb2-4277-b682-024eced4f399.png)

It seems that you have modified the batch size. Maybe you can change
```
 0.01, 100
```
to 
```
0.05, 20
```
or other combinations.

**Comment by a07913838438** [comment]
> > Thanks for your great projects and Thanks for share the code of your great work!
> > I do this
> > ```
> > steps=-1,500,40000,60000
> > scales=0.1,10,.1,.1
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > to
> > ```
> > steps=-1,500,40000,60000
> > scales=0.01,100,.1,.1
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > but The problem remains,I don't know how to do it to solve this problem.Can you give me some advice?
> > ![2021-08-24 10-39-11屏幕截图](https://user-images.githubusercontent.com/89434608/130547463-e09c0a32-7bb2-4277-b682-024eced4f399.png)
> 
> It seems that you have modified the batch size. Maybe you can change
> 
> ```
>  0.01, 100
> ```
> 
> to
> 
> ```
> 0.05, 20
> ```
> 
> or other combinations.

thank you very much,I have been solve this problem,because of the learning rate is very high.

**Comment by pujianjian** [comment]
Now, the loss value is not nan, but the proposal is 0.
![1629858673(1)](https://user-images.githubusercontent.com/48307115/130717011-46e579b5-baf2-4e34-810e-b173fd8fd1ae.png)

**Comment by Bohao-Lee** [comment]
> Now, the loss value is not nan, but the proposal is 0.
> ![1629858673(1)](https://user-images.githubusercontent.com/48307115/130717011-46e579b5-baf2-4e34-810e-b173fd8fd1ae.png)

After training a while, proposal won't be 0. It only occurs at the beginning of training.
